<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
<comment> Copyright (C) 2010 - 2016 by Pedro Mendes, Virginia Tech Intellectual </comment>
<comment> Properties, Inc., University of Heidelberg, and The University </comment>
<comment> of Manchester. </comment>
<comment> All rights reserved. </comment>

<comment> Copyright (C) 2008 - 2009 by Pedro Mendes, Virginia Tech Intellectual </comment>
<comment> Properties, Inc., EML Research, gGmbH, University of Heidelberg, </comment>
<comment> and The University of Manchester. </comment>
<comment> All rights reserved. </comment>

<comment> Copyright (C) 2003 - 2007 by Pedro Mendes, Virginia Tech Intellectual </comment>
<comment> Properties, Inc. and EML Research, gGmbH. </comment>
<comment> All rights reserved. </comment>

 <comment> All rights reserved. </comment>
 <class>ReactionsWidget1</class>
 <widget class="CopasiWidget" name="ReactionsWidget1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>235</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="caption" stdset="0">
   <string>Metabolite</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="TextLabel5">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Reaction</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="3">
    <widget class="MyLineEdit" name="LineEdit2"/>
   </item>
   <item row="1" column="2" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="CheckBox">
       <property name="text">
        <string>Reversible</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="mpMultiCompartment">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Multi Compartment</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="5">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="TextLabel6">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Rate Law</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QComboBox" name="ComboBox1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToMinimumContentsLength</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QToolButton" name="newKinetics">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Add Rate Law&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QToolButton" name="editKinetics">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Edit Rate Law&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="TextLabel6_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Rate Law Unit</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="TextLabel7">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Symbol Definition</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="2" colspan="3">
    <widget class="ParameterTable" name="table">
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="columnCount">
      <number>5</number>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Role</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Mapping</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Value</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Unit</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="6" column="0" colspan="5">
    <widget class="Line" name="mpLine1">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="TextLabel8">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Flux</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QLineEdit" name="LineEdit3">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="mpDefaultUnit">
       <property name="text">
        <string>Default</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="mpConcentrationUnit">
       <property name="text">
        <string>Concentration</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="mpAmountUnit">
       <property name="text">
        <string>Amount</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CopasiWidget</class>
   <extends>QWidget</extends>
   <header>copasi/UI/copasiWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MyLineEdit</class>
   <extends>QLineEdit</extends>
   <header>copasi/UI/MyLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>ParameterTable</class>
   <extends>QTableWidget</extends>
   <header>copasi/UI/parametertable.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>LineEdit2</tabstop>
  <tabstop>CheckBox</tabstop>
  <tabstop>mpMultiCompartment</tabstop>
  <tabstop>ComboBox1</tabstop>
  <tabstop>newKinetics</tabstop>
  <tabstop>editKinetics</tabstop>
  <tabstop>mpDefaultUnit</tabstop>
  <tabstop>mpConcentrationUnit</tabstop>
  <tabstop>mpAmountUnit</tabstop>
  <tabstop>table</tabstop>
  <tabstop>LineEdit3</tabstop>
 </tabstops>
 <includes>
  <include location="global">string</include>
  <include location="local">copasi/UI/copasiWidget.h</include>
  <include location="local">copasi/UI/MyLineEdit.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>ComboBox1</sender>
   <signal>activated(QString)</signal>
   <receiver>ReactionsWidget1</receiver>
   <slot>slotComboBoxSelectionChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LineEdit2</sender>
   <signal>edited()</signal>
   <receiver>ReactionsWidget1</receiver>
   <slot>slotLineEditChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CheckBox</sender>
   <signal>clicked()</signal>
   <receiver>ReactionsWidget1</receiver>
   <slot>slotCheckBoxClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpDefaultUnit</sender>
   <signal>clicked(bool)</signal>
   <receiver>ReactionsWidget1</receiver>
   <slot>slotDefaultUnitChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpConcentrationUnit</sender>
   <signal>clicked(bool)</signal>
   <receiver>ReactionsWidget1</receiver>
   <slot>slotConcentrationUnitChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpAmountUnit</sender>
   <signal>clicked(bool)</signal>
   <receiver>ReactionsWidget1</receiver>
   <slot>slotAmountUnitChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>newKinetics</sender>
   <signal>clicked()</signal>
   <receiver>ReactionsWidget1</receiver>
   <slot>slotNewFunction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editKinetics</sender>
   <signal>clicked()</signal>
   <receiver>ReactionsWidget1</receiver>
   <slot>slotGotoFunction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>336</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotCheckBoxClicked()</slot>
  <slot>slotNewFunction()</slot>
  <slot>slotComboBoxSelectionChanged(QString)</slot>
  <slot>slotLineEditChanged()</slot>
  <slot>slotGotoFunction()</slot>
  <slot>slotDefaultUnitChecked(bool)</slot>
  <slot>slotConcentrationUnitChecked(bool)</slot>
  <slot>slotAmountUnitChecked(bool)</slot>
 </slots>
</ui>
